openBCIBoard.js

```diff
@@ -206,9 +240,19 @@ function OpenBCIFactory() {
      * @author AJ Keller (@pushtheworldllc)
      */
     OpenBCIBoard.prototype.streamStart = function() {
-        this.streaming = true;
-        this._reset();
-        return this.write(k.OBCIStreamStart);
+        return new Promise((resolve, reject) => {
+            console.log('got here');
+            if(this.streaming) reject('Error [.streamStart()]: Already streaming');
+            this.streaming = true;
+            this._reset();
+            this.write(k.OBCIStreamStart)
+                .then(() => {
+                    setTimeout(() => {
+                        resolve();
+                    }, 50); // allow time for command to get sent
```

Comment:
timeout should probably be a constant. may be eliminate with the radio code down the line.